README.md

```diff
@@ -99,7 +119,11 @@ $("button").on("click", function(event){
 
 Your Answer:
 ```js
-// write code here
+var changeColor = function(){
+  document.addEventListener("click", function(){
+   document.getElementById("button").css("color", "red");
+ });
+ }
 ```
```

Comment:
Your code places an event listener on the entire document - not just a button(s). You'll need to use `querySelector` or `querySelectorAll` to retrieve the button(s) for you, like so...

```js
document.querySelector("button")
// Note that we are selecting by the tag name, not an ID.
```

The final code would look something like this...

```js
document.querySelector("button").addEventListener("click", function(event){
  event.target.style.color = "red";
});
```